pkg/cloudprovider/provider/vsphere/provider.go

```diff
@@ -164,6 +168,11 @@ func (p *provider) getConfig(provSpec clusterv1alpha1.ProviderSpec) (*Config, *p
 		return nil, nil, nil, err
 	}
 
+	c.Cluster, err = p.configVarResolver.GetConfigVarStringValue(rawConfig.Cluster)
```

Comment:
The comment for this field in the raw config type is:

```
// Cluster is a noop field, it's not used anywhere but left here intentionally for backward compatibility purposes
```

Can we update this, since it's not true anymore?